tests/h/storage_test.py

```diff
@@ -126,624 +125,298 @@ def test_expand_uri_document_uris(self, db_session, normalized, expected_uris):
         assert uris == expected_uris
 
 
-@pytest.mark.usefixtures("models")
```

Comment:
OMG. Just no.

I basically had to start over with these. There was no way to incrementally make them be what we want.